hacking/aws_config/test_policies/container-policy.yaml

```diff
@@ -0,0 +1,39 @@
+Version: '2012-10-17'
+Statement:
+  - Sid: AllowGlobalUnrestrictedResourceActionsWhichIncurNoFees
+    Effect: Allow
+    Action:
+      - application-autoscaling:Describe*
+      - application-autoscaling:PutScalingPolicy
+      - application-autoscaling:RegisterScalableTarget
+      - ecs:DeregisterTaskDefinition
+      - ecs:CreateService
+      - ecs:DeleteService
+      - ecs:Describe*
+      - ecs:List*
+      - ecs:PutAccountSetting
+      - ecs:RegisterTaskDefinition
+      - ecs:UpdateService
+      - elasticloadbalancing:Describe*
+    Resource: "*"
+  - Sid: AllowGlobalUnrestrictedResourceActionsWhichIncurFees
+    Effect: Allow
+    Action:
+      - ecs:CreateCluster
+      - ecs:DeleteCluster
+      - ecs:RunTask
+      - ecs:StartTask
+      - ecs:StopTask
+    Resource: "*"
+  #- Sid: AllowGlobalResourceRestrictedActionsWhichIncurNoFees
+  #  Effect: Allow
```

Comment:
What's the purpose of these commented out unrestricted policies, here and in the devops, networking, and storage policies below?

Reply:
I'm hoping that if we leave them there commented out, then should someone need/want to add some we'll get a consistent naming of the policies.